roles/foreman_setup/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+foreman_setup_foreman_host: "{{ ansible_fqdn }}"
+foreman_setup_foreman_url: "https://{{ foreman_setup_foreman_host }}/api/v2/hosts"
+foreman_setup_data:
+  host:
+    name: "{{ ansible_fqdn }}"
+    location_id: 2
+    organization_id: 1
```

Comment:
This is not great, we've seen it changed recently and then switched back again.

Reply:
Fair point. Registration is hard. Let's make an issue to resolve that since this already exists.